components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -426,25 +404,19 @@ function matchFunctions.getMapVeto(match)
 		end
 	end
 	if data[1] then
-		data[1].vetostart = vetostart
+		data[1].vetostart = vetoStart
 	end
 	return data
 end
 
 -- Parse MVP input
 function matchFunctions.getMVP(match)
 	if not match.mvp then return nil end
-	local mvppoints = match.mvppoints or 1
 
-	-- Split the input
-	local players = mw.text.split(match.mvp, ',')
-
-	-- Trim the input
-	for index,player in pairs(players) do
-		players[index] = mw.text.trim(player)
-	end
+	-- Split & trim the input
+	local players = Table.mapValues(mw.text.split(match.mvp, ','),mw.text.trim)
 
-	return {players=players, points=mvppoints}
+	return {players = players, points = match.mvppoints or 1}
```

Comment:
maybe when cleaning it up use `MatchGroupInput.readMvp(match)`
(adjusts storage though so the mvpTable module needs adjusting too, see e.g. lol for an example)

Reply:
Leave that for another time, as it would require more adjustments. But it's something that's on my radar